mne/preprocessing/tests/test_ica.py

```diff
@@ -608,4 +608,27 @@ def test_fit_params():
     ICA(fit_params=fit_params)  # test no side effects
     assert_equal(fit_params, {})
 
+
+@requires_sklearn
+def test_bad_channels():
```

Comment:
Sorry, I meant EOG to be *included*. You should add a test that makes sure you can include EOG channels in the ICA process, since that's what this PR adds (right?). At the very least just make sure it works without throwing an error. This is the type of test that would throw an error on `master` currently, so by including it you're working on TDD (test-driven development) -- although strictly speaking I think such a test should be the *first* change, not the last, to qualify as TDD. But in practice we rarely do it that way :)